### ISSUE TITLE
Smart-accordion keyboard accessibility error

### ISSUE THREAD

**Issue by timmhayes** [HTMLElements/smart-webcomponents#8]
The smart-accordion component should be accessible via keyboard for mobility impaired users. And while you do have code in the component to handle keyboard navigation, it throws an error when used.

**To Reproduce**
1. Create a smart-accordion component
2. Tab to one of the accordion panels
3. Click click the 'Enter' key to open the panel with the keyboard
4. Panel does not open as expected and code throws "Uncaught TypeError: Cannot read property 'expanded' of undefined" error.

**Expected behavior**
Panel opens with keyboard click.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Chrome
 - Version 87

**Additional context**
I know this can be fixed by adding custom CSS, but your out-of-the-box styles fail WCAG rule 2.4.7 for showing keyboard focus. Please see  https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-visible.html

**Comment by bmarkov** [comment]
We tried to reproduce that in our demo here: https://www.htmlelements.com/demos/accordion/overview/. Unfortunately, we were unable to reproduce it.  It was possible to tab and shift+tab through the panels and Enter key opens/closes the focused panel.

**Comment by timmhayes** [comment]
Strange... I still see it break on that page. 

I used this CSS to provide a focus indicator (just added in Chrome's dev tools). 
```
smart-accordion-item:focus {
  outline: 2px black solid;
}
```
As I mentioned before, users would need to add something like this to make the accordion WCAG-compliant. And since you provide a tabindex to the _smart-accordion-item_ element, I assumed that was the right element to use. Is that the element you are opening/closing with the Enter key?

Full error details:
```
/demos/source/modules/smart.accordion.js:8 Uncaught TypeError: Cannot read property 'expanded' of undefined
    at HTMLElement.value (/demos/source/modules/smart.accordion.js:8)
    at HTMLElement.<anonymous> (/demos/source/modules/smart.accordion.js:8)
    at e.value (/demos/source/modules/smart.accordion.js:8)
```

**Comment by bmarkov** [comment]
Tabbing through the accordion changes the Background of the Focused Item. We do not provide a "tabindex" property, reference: https://www.htmlelements.com/docs/accordion-api/. Tabbing and keyboard navigation is handled by the accordion component.

**Comment by timmhayes** [comment]
I don't see a change to the background of the accordion when tabbing, which is why I manually added an outline in CSS. Is it supposed to show a visual difference when focused? Are you seeing that on the demo page you referenced above?  I don't see that, which makes it hard to navigate using a keyboard.... aside from the script error.

I know the Accordion component does not have a tabindex property in its API. I was only reading the code of what the component rendered so I could see what was broken.

I don't understand why you would be seeing something different from me on the demo page, since I'm doing this on a standard version of Chrome on Windows 10. And I just checked Firefox, it fails in exactly the same way with a slightly different error message:
```
Uncaught TypeError: i is undefined
    value https://www.htmlelements.com/demos/source/modules/smart.accordion.js:8
    i https://www.htmlelements.com/demos/source/modules/smart.accordion.js:8
    value https://www.htmlelements.com/demos/source/modules/smart.accordion.js:8
```

**Comment by bmarkov** [comment]
I would suggest clearing the browser history and refreshing the page. We cannot reproduce the reported behavior and we see a different background when tabbing through accordion items(see the image below)

![accordion](https://user-images.githubusercontent.com/20238579/104554961-b4252f00-5645-11eb-8626-5f403597a19e.png)

**Comment by timmhayes** [comment]
I can verify that it is broken for me in Edge, Chrome, Chrome Dev, and Firefox on two different computers. And I started shooting a screencast to show you. But then I realized I could actually get the accordion to work with the keyboard if I **clicked on one of the tabs first with a mouse**. I think you may have done that when trying to reproduce the issue. So, new steps to reproduce:

1. Open https://www.htmlelements.com/demos/accordion/overview/
2. Using only the keyboard (never clicking the accordion tab with your mouse), navigate to the accordion. You can click the header "Creation flows" to set the focus closer to the accordion if you want. 
3. Tab into the panel - you will not see focus.
4. Hit enter key - you will see an error.

If you follow these steps, I'm pretty sure you will be able to reproduce the issue.

**Comment by bmarkov** [comment]
Hi Tim,

We confirm that. Testing it we were clicking and then tabbing through the items.
